### DISCUSSION TITLE
Hope to join the support for tabulator!

### DISCUSSION THREAD

**Discussion by sx98083714** [zauberzeug/nicegui#2975] [Ideas / Feature Requests]
I've found an excellent table component, tabulator (https://tabulator.info/), which is a lot more powerful for ui.table and just as powerful as aagrid. Personal suggestions can be removed from the nicegui to change to tabulator), I refer to the official website of the custom vue component example, can achieve some basic functions, such as the implementation of grouping, sorting, filtering, moving, etc., which has been too powerful, but I have always been unable to call some of the built-in methods in tabulator, Such a good open source project, after careful comparison, you will find that tabulator is how powerful and excellent an open source project, excellent performance, very many features, can meet almost all kinds of needs, so such a good open source project, why not let it join nicegui, so that nicegui become more powerful and more general better?

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @sx98083714!

[Tabulator](https://tabulator.info/) looks great and might be a great addition to NiceGUI. To prevent us from maintaining yet another rather complex table element, we think it might be best to develop it in a separate repository and PyPI package, like we did for Highcharts: https://github.com/zauberzeug/nicegui-highcharts. But we need to find a maintainer from the community, since we're already pretty busy with existing UI elements and NiceGUI's core features.

**Comment by falkoschindler** [comment]
Looks like @CrystalWindSnake implemented such an element:
https://github.com/CrystalWindSnake/nicegui-tabulator

**Reply by falkoschindler** [reply]
Oh, it has already been mentioned here: https://github.com/zauberzeug/nicegui/discussions/1148.
